buildlib/pr/go/go-test.yml

```diff
@@ -64,9 +65,6 @@ jobs:
         az_module_load dev/go-latest
         go_port=$((30000 + $(AZP_AGENT_ID) * 100))
         args="-p=$go_port"
-        if [ "${{ parameters.name }}" == "gpu" ]; then
```

Comment:
pls comment out this code instead of removing it, and add a comment:
`# FIXME Temporarily disabled GPU tests due to cuda_ipc device context issue`